File: abc/abc046/a.py
# https://atcoder.jp/contests/abc046/submissions/35805842

a, b, c = map(int, input().split())
a, b, c = sorted([a, b, c])
if a == b == c: print(1)
elif a == b or b == c: print(2)
else: print(3)

File: abc/abc046/a.cpp
// https://atcoder.jp/contests/abc046/submissions/26606471

#include <bits/stdc++.h>
using namespace std;
int main() {
  int a, b, c;
  cin >> a >> b >> c;
  if(a == b && b == c) cout << 1;
  else if((a == b && a != c) || (a != b && a == c)  || (b == c && a != b)) cout << 2;
  else cout << 3;
}
